src/main/resources/io/jenkins/plugins/designlibrary/Localization/index.jelly

```diff
@@ -68,5 +68,62 @@ THE SOFTWARE.
         </s:code-panes>
       </s:group>
     </s:section>
+    <s:section title="${%titlejs}">
+      <h3 class="jdl-section__subtitle">
```

Comment:
I'd probably just have these as separate sections, e.g. `Localize text in JavaScript via Jelly` and `Localize text in JavaScript via Java`